.github/workflows/platform-frontend-ci.yml

```diff
@@ -29,15 +29,11 @@ jobs:
 
       - name: Install dependencies
         run: |
-          npm install
-
-      - name: Check formatting with Prettier
-        run: |
-          npx prettier --check .
+          yarn install --frozen-lockfile
```

Comment:
This fails when `yarn.lock` is inconsistent with `package.json`

Reply:
This is nice, thanks!